packages/gateway/plugins/admin-context-plugin.ts

```
@@ -1,15 +1,74 @@
 import { useExtendContext } from 'graphql-yoga';
+import NodeCache from 'node-cache';
 import pg from 'pg';
 import { sql } from '@pgtyped/runtime';
-import type { Currency } from '@shared/enums';
-import { formatCurrency, getCacheInstance } from '@shared/helpers';
-import type { Environment } from '@shared/types';
-import { env } from '../environment.js';
-import type {
+import { env } from '../../../packages/server/src/environment';
```

ESLint

[error] Missing file extension for "../../../packages/server/src/environment"

(import/extensions)